tests/forms_tests/tests/test_forms.py

```diff
@@ -3773,3 +3773,24 @@ class CustomForm(Form):
         custom = CustomRenderer()
         form = CustomForm(renderer=custom)
         self.assertEqual(form.renderer, custom)
+
+    def test_css_classes(self):
+        form = Person()
+        self.assertEqual(form['first_name'].css_classes(None), '')
+        self.assertEqual(form['first_name'].css_classes(), '')
+        self.assertEqual(form['first_name'].css_classes(''), '')
+        self.assertEqual(form['first_name'].css_classes('mb-0'), 'mb-0')
+        self.assertEqual(form['first_name'].css_classes('mb-0 mb-0'), 'mb-0')
+
+        form.required_css_class = 'required'
+        form.error_css_class = 'error'
+        self.assertEqual(form['first_name'].css_classes(), 'required')
+
+        form.fields['first_name'].required = False
+        self.assertEqual(form['first_name'].css_classes(), '')
+
+        form = Person({})
+        self.assertEqual(form['first_name'].css_classes(), '')
+        form.error_css_class = 'error'
+        css = form['first_name'].css_classes()
+        self.assertIn('error', css)
```

Comment:
This is already tested in `test_error_html_required_html_classes()` and `test_field_with_css_class()`. I'm going to remove these lines.